Portable_PDD/Compile.bat

```diff
@@ -5,11 +5,14 @@ set DOCUMENT_PATH="markdown_template.md"
 REM This is where you specify the final name of your document.  Spaces will be replaced with underscores.
 set FINAL_DOC_NAME="50_SAT"
 
+REM This is the where you specify the template of the .tex file.
+set PDD_TEMPLATE="spex_template.tex"
+
 REM You can specify the type of compiler script.  It expects an input of the path of the .tex document.
-set COMPILE_SCRIPT="Compile_Scripts\\DockerCompiler.bat"
+set COMPILE_SCRIPT="Compile_Scripts\\MikTexCompiler.bat"
```

Comment:
When you merged you relocated the two folders so this path needs to change to ..\Compile_Scripts